src/messages/proposals.rs

```diff
@@ -169,11 +194,11 @@
 /// Proposal queue that helps filtering and sorting the Proposals from one
 /// epoch.
 #[derive(Default)]
-pub struct ProposalQueue {
-    queued_proposals: HashMap<ProposalID, QueuedProposal>,
+pub struct ProposalQueue<'a> {
```

Comment:
`pub(crate)`?

Reply:
It's publicly exposed in `apply_commit` unfortunately.